README.md

```diff
@@ -42,6 +42,8 @@ Dart Code can be [installed from the Visual Studio Code Marketplace](https://mar
 
 - `dart.allowAnalytics`: Note: We only send a few very basic events and the platform and extension/Dart version numbers :-)
 - `dart.checkForSdkUpdates`: Whether to check you are using the latest version of the Dart SDK at startup. Defaults to `true`.
+- `dart.debugSdkLibraries`: Whether SDK libraries should be marked as debuggable. Defaults to `false`.
```

Comment:
My 0.02 would be to default these to `true` (debuggable), as that would be more familiar to how I work. You milage my vary :)